CMP1903_A1_2324/Die.cs

```diff
@@ -15,8 +15,15 @@ internal class Die
 
         //Property
 
-
+        public int CurrentValue { get; private set; }
```

Comment:
Consider making some variables private to reduce the chances of unwanted influence on them. Ensure that if done, you follow the C# style guidelines ( _camelCase ).

Reply:
CurrentValue cannot be made private due to the nature of it needing to be accessed outside the class, however appropriate getters and setters were introduced to allow for the variable to remain encapsulated.